Makefile.custom.mk

```diff
@@ -12,8 +12,15 @@ clean: ## Clean the git work dir and remove all untracked files
 ##@ Testing
 
 .PHONY: test
-test: install-tools ## run unit tests for alerting rules
+test: install-tools template-chart test-rules
+
+test-rules: install-tools template-chart
+	# run unit tests for alerting rules
 	bash test/hack/bin/verify-rules.sh "$(test_filter)"
 
 install-tools:
 	./test/hack/bin/fetch-tools.sh
+
+template-chart: install-tools
+	# prepare the helm chart
+	test/hack/bin/architect helm template --dir helm/prometheus-rules --dry-run
```

Comment:
For some cases, one will have to specify a directory to which direct the output of the command (with the `--output-directory` option). Is it possible to make this configurable ?